server/radar.py

```diff
@@ -49,18 +52,40 @@ def merge_first_and_last_names(c: Counter) -> Counter:
 
 
 def headlines_by_source(source: str, start_date=None, end_date=None) -> list[str]:
-    query = supabase_client.table("top-headlines-news").select("title, source_id").filter("source_id", "eq", source)
+    query = supabase_client.table("top-headlines-news").select("title, source_id, content").filter("source_id", "eq", source)
```

Comment:
`top-headlines-news` table is deprecated. That was the one that scraped data from the news API every hour. We use the `homepage-news` table which gets data from our individual website scrapers.

Does this query need to be updated?

Reply:
Whoops, yeah let me update that.